firstvoices/backend/management/commands/copy_site.py

```diff
@@ -0,0 +1,581 @@
+        .distinct()
+    )
+    for entry in dictionary_entries:
+        source_categories = list(entry.categories.all())
+        source_related_characters = list(entry.related_characters.all())
+        source_related_entries = list(entry.related_dictionary_entries.all())
+        source_media = {
+            "audio": list(entry.related_audio.values_list("id", flat=True)),
+            "images": list(entry.related_images.values_list("id", flat=True)),
+            "videos": list(entry.related_videos.values_list("id", flat=True)),
+        }
+
+        source_entry_id = entry.id
+        target_entry_id = uuid.uuid4()
+        dictionary_entry_map[source_entry_id] = target_entry_id
+
+        entry.id = target_entry_id
+        entry.site = target_site
+        entry.batch_id = ""
+        entry.import_job = None
+        entry.save()
+
+        updated_categories = [
+            category_map[category.id] for category in source_categories
+        ]
+        entry.categories.set(updated_categories)
+
+        updated_related_characters = [
+            character_map[char.id] for char in source_related_characters
+        ]
+        entry.related_characters.set(updated_related_characters)
+
+        updated_related_entries = [
+            dictionary_entry_map[source_related_entry.id]
+            for source_related_entry in source_related_entries
+            if source_related_entry.id in dictionary_entry_map
+        ]
+        entry.related_dictionary_entries.set(updated_related_entries)
+
+        copy_related_media(entry, source_media, audio_map, image_map, video_map)
+        entry.save()
+
+    return dictionary_entry_map
+
+
+def copy_immersion_labels(source_site, target_site, dictionary_entry_map):
+    imm_labels = list(ImmersionLabel.objects.filter(site=source_site))
+    for imm_label in imm_labels:
+        imm_label.id = uuid.uuid4()
+        imm_label.site = target_site
+
+        source_dictionary_entry = imm_label.dictionary_entry
+        imm_label.dictionary_entry_id = dictionary_entry_map[source_dictionary_entry.id]
+
+        imm_label.save()
+
+
+def copy_related_objects(source_site, target_site, user, logger):
+    copy_site_features(source_site, target_site)
+    logger.info("Site features copied.")
+
+    category_map = copy_categories_and_return_map(source_site, target_site, user)
+    logger.info("Categories copied.")
+
+    audio_map = copy_audio_and_speakers_and_return_map(source_site, target_site, logger)
+    logger.info("Audio and speakers copied.")
+    image_map = copy_images_and_return_map(source_site, target_site, logger)
+    logger.info("Images copied.")
+    video_map = copy_videos_and_return_map(source_site, target_site, logger)
+    logger.info("Videos copied.")
+
+    character_map = copy_all_characters_and_return_map(
+        source_site, target_site, audio_map, image_map, video_map
+    )
+    copy_alphabet(source_site, target_site)
+    logger.info("Characters and alphabet copied.")
```

Comment:
[minor] should all the "___ copied" debug statements be separate to their own category? (characters, alphabet, songs, stories etc.)